rustworkx/graph.pyi

```diff
@@ -22,14 +24,19 @@ from typing import (
     List,
     Tuple,
     Sequence,
+    TYPE_CHECKING,
 )
 
+if TYPE_CHECKING:
+    from .digraph import PyDiGraph
+
```

Comment:
PyGraph and PyDiGraph have a "cyclic" dependency due to the `to_directed` and `to_undirected` method. This is a hack to make Python agree with the Rust code